newsfragments/2431.misc.rst

```diff
@@ -0,0 +1 @@
+Introduces a probationary period for policy creation in the network, until 2021-01-31 23:59:59 UTC.
```

Comment:
Update the newsfragment to the final date

Reply:
:heavy_check_mark: